src/api/schema/metrics/events_in.rs

```diff
@@ -0,0 +1,96 @@
+use crate::event::{Metric, MetricValue};
+use async_graphql::Object;
+use chrono::{DateTime, Utc};
+
+pub struct EventsInTotal(Metric);
+
+impl EventsInTotal {
+    pub fn new(m: Metric) -> Self {
+        Self(m)
+    }
+
+    pub fn get_timestamp(&self) -> Option<DateTime<Utc>> {
+        self.0.data.timestamp
+    }
+
+    pub fn get_events_in_total(&self) -> f64 {
+        match self.0.data.value {
+            MetricValue::Counter { value } => value,
+            _ => 0.00,
```

Comment:
I'm still a little concerned about conflating `0` for components that will never publish this metric. I could see users being surprised that it never changes. This concern exists with the existing `events_processed_total`s though, so I'm ok with following up on this.

Reply:
Follow up sounds good, and once we start using `events_in_*` we will be able to remove `events_processed_totals` which will make this simpler.

Reply:
Yeah, if we change this to return an `Option<f64>`, it should probably be done everywhere. I'm okay with leaving this in too for now to mimic current behaviour.